L1Trigger/TrackFindingTracklet/src/TrackDerTable.cc

```diff
@@ -0,0 +1,1478 @@
+            (1 << settings->fitphi0bitshift()) * MinvDt[1][2 * i + 1] * settings->kzproj() / settings->kphi0pars();
+        iMinvDt[2][2 * i + 1] =
+            (1 << settings->fittbitshift()) * MinvDt[2][2 * i + 1] * settings->kzproj() / settings->ktpars();
+        iMinvDt[3][2 * i + 1] =
+            (1 << settings->fitz0bitshift()) * MinvDt[3][2 * i + 1] * settings->kzproj() / settings->kzpars();
+      } else {
+        int fact = (1 << (settings->nzbitsstub(0) - settings->nzbitsstub(5)));
+
+        iMinvDt[0][2 * i + 1] = (1 << settings->fitrinvbitshift()) * MinvDt[0][2 * i + 1] * fact * settings->kzproj() /
+                                settings->krinvpars();
+        iMinvDt[1][2 * i + 1] = (1 << settings->fitphi0bitshift()) * MinvDt[1][2 * i + 1] * fact * settings->kzproj() /
+                                settings->kphi0pars();
+        iMinvDt[2][2 * i + 1] =
+            (1 << settings->fittbitshift()) * MinvDt[2][2 * i + 1] * fact * settings->kzproj() / settings->ktpars();
+        iMinvDt[3][2 * i + 1] =
+            (1 << settings->fitz0bitshift()) * MinvDt[3][2 * i + 1] * fact * settings->kzproj() / settings->kzpars();
+      }
+    }
+
+    //Secondly the disks
+    else {
+      if (std::abs(alpha[i - nlayers]) < 1e-10) {
+        MinvDt[0][2 * i] *= (rnew[i]);
+        MinvDt[1][2 * i] *= (rnew[i]);
+        MinvDt[2][2 * i] *= (rnew[i]);
+        MinvDt[3][2 * i] *= (rnew[i]);
+      } else {
+        MinvDt[0][2 * i] *= (rnew[i]);
+        MinvDt[1][2 * i] *= (rnew[i]);
+        MinvDt[2][2 * i] *= (rnew[i]);
+        MinvDt[3][2 * i] *= (rnew[i]);
+      }
+
+      assert(MinvDt[0][2 * i] == MinvDt[0][2 * i]);
+
+      iMinvDt[0][2 * i] =
+          (1 << settings->fitrinvbitshift()) * MinvDt[0][2 * i] * settings->kphiproj123() / settings->krinvpars();
+      iMinvDt[1][2 * i] =
+          (1 << settings->fitphi0bitshift()) * MinvDt[1][2 * i] * settings->kphiproj123() / settings->kphi0pars();
+      iMinvDt[2][2 * i] =
+          (1 << settings->fittbitshift()) * MinvDt[2][2 * i] * settings->kphiproj123() / settings->ktpars();
+      iMinvDt[3][2 * i] =
+          (1 << settings->fitz0bitshift()) * MinvDt[3][2 * i] * settings->kphiproj123() / settings->kz();
+
+      iMinvDt[0][2 * i + 1] = (1 << settings->fitrinvbitshift()) * MinvDt[0][2 * i + 1] * settings->krprojshiftdisk() /
+                              settings->krinvpars();
+      iMinvDt[1][2 * i + 1] = (1 << settings->fitphi0bitshift()) * MinvDt[1][2 * i + 1] * settings->krprojshiftdisk() /
+                              settings->kphi0pars();
+      iMinvDt[2][2 * i + 1] =
+          (1 << settings->fittbitshift()) * MinvDt[2][2 * i + 1] * settings->krprojshiftdisk() / settings->ktpars();
+      iMinvDt[3][2 * i + 1] =
+          (1 << settings->fitz0bitshift()) * MinvDt[3][2 * i + 1] * settings->krprojshiftdisk() / settings->kz();
+    }
+  }
+}
+
+double TrackDerTable::gett(const Settings* settings, int diskmask, int layermask) {
```

Comment:
this function could have a more meaningful name

Reply:
it gets the track parameter "t" => gett(), this seems meaningful enough to me?

Reply:
i have now renamed this to tpar() instead to also be better consistent with avoiding getXYZ() in favor of XYZ()